frontend/src/pages/CaptionGenerator.tsx

```
@@ -0,0 +1,104 @@
+        >
+          <option value="casual">Casual</option>
+          <option value="formal">Formal</option>
+          <option value="promotional">Promotional</option>
+        </select>
+      </div>
+
+      <button
+        onClick={handleGenerate}
+        disabled={loading}
+        className="px-4 py-2 bg-black text-white rounded"
+      >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)